test/helpers/env.js

```diff
@@ -203,6 +214,7 @@ env.IOS82 = env.DEVICE.match(/ios82/i);
 env.IOS83 = env.DEVICE.match(/ios83/i);
 env.IOS84 = env.DEVICE.match(/ios84/i);
 env.IOS9 = env.DEVICE.match(/ios9/i);
+env.IOS91 = env.DEVICE.match(/ios91/i);
```

Comment:
The same functionality is in place for iOS7 & iOS8. @moiz can you confirm there are no _iOS9.0 only_ features?